pages/doc/wavefront_prometheus.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Prometheus Queries in Wavefront
+keywords: query language
+sidebar: doc_sidebar
+published: true
+permalink: wavefront_prometheus.html
+summary: Run Prometheus from the Wavefront query line
```

Comment:
This line sounds strange because when the word "Prometheus" it typically means "Prometheus Server" which is a service which runs on a customer's cluster.  Also "Wavefront query line" sounded weird to me.  I would have expected it to be called the "query editor" (this comment applies to anywhere else we use "query line")

Reply:
Fixed. Now using Query Editor.